Meeting-Material/readme.md

```diff
@@ -0,0 +1,33 @@
+# Guide for meeting chairs
+This guide is meant to help to keep the effort as low as possible for the meeting chair while keeping the documentation as accurate as possible at the same time.
+
+## Before the Meeting 
+1. create a preparation branch "meeting_yyyymmdd" with yyyymmdd the date of the next meeting
+2. in the branch create a new file in a respective subfolder /Meeting-yyyymmdd/yyyymmdd.md
+3. you can use the following file [tbd] as template for the content of the new .md meeting log
+4. Check the mailing list for topics relevant for the meeting - potentially request topics by sending a mail to the list (may also serve as reminder)
+5. Prepare the standard content including the agenda
+6. Prepare a web-conference system (ideally BigBlueButton https://conf.fsfe.org/b/compliance-tooling see hints below)
+7. (optional) Check the OpenChain Global Calendar if the meeting details are correctly presented there and if necessary contact him for update
+8. Create a meeting invitation including the agenda and send it to the mailing list 
+
+## While the Meeting 
+1. Start the meeting in the respective meeting tool (ideally Big Blue Button)
+2. (optional) Ask / inform all participants for consent for recording and turn the cameras off. Start recording
+3. Show the Antitrust slide from OpenChain and inform about the Antitrust requirements (e.g. "Stick to the shared agenda", ...). 
```

Comment:
Can you please add the link to the slide and if it is not available in the repo can you please add it to the repo?